### ISSUE TITLE
NullPointerException for getText method of ListItemContext class

### ISSUE THREAD

**Issue by Xiaoshu-Zhao** [opensagres/xdocreport#637]
`getText`'s code is following：
```
public String getText() {
        String text = this.lvl.getLvlText().getVal();
        CTNumFmt numFmt = this.lvl.getNumFmt();
        if (!STNumberFormat.BULLET.equals(numFmt)) {
            List<String> numbers = new ArrayList();

            for (ListItemContext item = this; !item.isRoot(); item = item.getParent()) {
                numbers.add(0, item.getNumberText());
            }

            String num = null;

            for (int i = 0; i < numbers.size(); ++i) {
                if (numbers.get(i) instanceof String) {
                    num = (String) numbers.get(i);
                }
                text = StringUtils.replaceAll(text, "%" + (i + 1), num);
            }
        }

        return text;
    }
```
`String text = this.lvl.getLvlText().getVal();` may occur **NullPointerException** because `this.lvl.getLvlText()` can be null. I suggest add following code at beginning to avoid **NullPointerException**.

```
if (this.lvl == null || this.lvl.getLvlText() == null) {
    return "";
}
```

**Comment by Xiaoshu-Zhao** [comment]
Hi, I just create a pr, let me know if there is anything I need to do to get it approve. It is my first time to create a pr for open source repo lol.

**Comment by Adriansun** [comment]
Will this fix be released soon? A version 2.0.5 with a fix for the PDF problem, which seem to be the same problem that this one fix, would be great.
